res/css/views/rooms/_EventTile.scss

```diff
@@ -847,8 +844,8 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
     }
 
     .mx_MessageTimestamp {
-        max-width: 80px;
-        width: auto;
+        font-size: $font-12px;
+        max-width: var(--MessageTimestamp-max-width);
```

Comment:
`$MessageTimestamp_width` is enabled here.

|Before|After|
|---------|------|
|![before](https://user-images.githubusercontent.com/3362943/173214682-5ea3127d-b5d3-4c51-a536-3d5acd60ea2f.png)|![after](https://user-images.githubusercontent.com/3362943/173214681-a052cbcc-46f1-48eb-80e5-c6001ce0cc3a.png)|